src/components/Input/index.tsx

```diff
@@ -88,7 +88,12 @@ const Input =
 
         {(required && !value) && <RequiredIndicator reversedTheme={reversedTheme}>*</RequiredIndicator>}
 
-        {(!displayError && value && displayName) && <DisplayName reversedTheme={reversedTheme}> * {displayName} </DisplayName> }
+        {
+          (!displayError && value && displayName) && 
+          <DisplayName reversedTheme={reversedTheme}> 
+            {`${required ? '*' : ''} ${displayName}`.trim()}
```

Comment:
Maybe do it like this, without trim?
```{`${required ? '* ' : ''}${displayName}`}```

Reply:
nope, I want it to be
`* Display name`, not
`*Display name`

Reply:
uh, you hid space in asterisk string. well, same shit, idk if it worth changing it

Reply:
Yes,  there is a space after '* '